pkg/lockfile/parse-nuget-lock.go

```diff
@@ -0,0 +1,64 @@
+package lockfile
+
+import (
+	"encoding/json"
+	"fmt"
+	"os"
+)
+
+type NuGetLockPackage struct {
+	Resolved string `json:"resolved"`
+}
+
+type NuGetLockfile struct {
+	Version      int                                    `json:"version"`
```

Comment:
Is there a reference to the spec that we can leave as a comment here?

Also curious if we should be checking the "version" here.